Source/common/santa.proto

```diff
@@ -435,11 +432,15 @@ message SantaMessage {
   // Can allow client to relate events across time
   optional string uuid = 1;
 
+  // Machine ID of the host emitting this log
+  // Only valid when EnableMachineIDDecoration configuration option is set
+  optional string machine_id = 2;
+
   // Timestamp when the event occurred
-  optional google.protobuf.Timestamp event_time = 2;
+  optional google.protobuf.Timestamp event_time = 3;
```

Comment:
I assume this proto isn't currently being used by anything?

Reply:
Correct.